src/main/resources/archetype-resources/silo/src/main/resources/db/common/V1__schema.sql

```diff
@@ -0,0 +1,4 @@
+-- THIS IS AN EXAMPLE - THINK BEFORE YOU USE IT
```

Comment:
Technically if there is V1 in common the min. index for migration/test is V2 imho.